binding.cc

```diff
@@ -1827,7 +1800,6 @@ NAPI_INIT() {
 
   NAPI_EXPORT_FUNCTION(iterator_init);
   NAPI_EXPORT_FUNCTION(iterator_seek);
-  NAPI_EXPORT_FUNCTION(iterator_end);
```

Comment:
On the perspective of the level-ecosystem, we cannot expect iterators of all kinds of backends to end on GC automatically without a problem, it is implementation specific. Maybe later we should mark `iterator.end` as optional in `abstract-leveldown`?

Reply:
Nothing has to change in `abstract-leveldown` for now. I first want to explore the idea in `leveldown`.